plugins/nanoleaf/README.md

```diff
@@ -0,0 +1,23 @@
+Nanoleaf Plugin
+============
+
+This plugin creates a device per IP, each with one output port. Each port
```

Comment:
Could you add a link to the product.

Reply:
Will do, https://nanoleaf.me/en/consumer-led-lighting/products/smarter-series/nanoleaf-light-panels-smarter-kit/ if you fancy buying some :smile: